<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1211</width>
    <height>971</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SKA-LAB Tool</string>
  </property>
  <widget class="QTabWidget" name="qtabMain">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>1191</width>
     <height>956</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="qtabSkalab">
    <attribute name="title">
     <string>SkaLab</string>
    </attribute>
    <widget class="QComboBox" name="qcombo_profiles">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Default</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5xTPM</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="qlabel_profiles">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>SkaLab Profiles</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="qbutton_profile_load">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>230</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
    <widget class="QPushButton" name="qbutton_profile_delete">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>230</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="qradio_autosave">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string> Autosave on Exit</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="qradio_autoload">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>280</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string> AutoLoad on Program Start</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="qline_profile">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>480</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="qbutton_profile_saveas">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>410</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Save As...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="qbutton_profile_save">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>360</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QLabel" name="qlabel_profile">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>480</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Profile File:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="qlabel_profile_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>360</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Save changes in the current SkaLab Profile</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="qlabel_profile_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>410</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Save current setup in a New SkaLab Profile</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="qwpics" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>491</width>
       <height>122</height>
      </rect>
     </property>
    </widget>
    <widget class="QFrame" name="frame_station">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>20</y>
       <width>661</width>
       <height>891</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <widget class="QLabel" name="qlabel_config">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Station File</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="qline_configfile">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>  Browse and select a STATION config file</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="qbutton_load_configuration">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>30</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Load</string>
      </property>
     </widget>
     <widget class="QPushButton" name="qbutton_browse">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>30</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QPushButton" name="qbutton_station_init">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>80</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>INIT</string>
      </property>
     </widget>
     <widget class="QLabel" name="qlabel_bitfile">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>80</y>
        <width>421</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="qlabel_station">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>STATION</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QTableWidget" name="qtable_tpm">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>430</y>
        <width>611</width>
        <height>141</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>218</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>&quot;LOAD&quot; A STATION FILE</string>
       </property>
      </column>
     </widget>
     <widget class="QTableWidget" name="qtable_station">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>611</width>
        <height>201</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>308</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>&quot;LOAD&quot; A STATION FILE</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="qlabel_tpms">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>390</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>TPMs  -  Delays</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="qlabel_network">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>590</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>NETWORK</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QTableWidget" name="qtable_network">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>630</y>
        <width>611</width>
        <height>241</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>365</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>&quot;LOAD&quot; A STATION FILE</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="qlabel_bitfile_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>BITFILE: </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QTableWidget" name="qtable_profile">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>550</y>
       <width>461</width>
       <height>351</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>365</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>&quot;LOAD&quot; A SKALAB PROFILE</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="qlabel_connection_3">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>10</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(238, 238, 236);</string>
     </property>
     <property name="text">
      <string>STATION  CONFIGURATION</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="qtabSubrack">
    <attribute name="title">
     <string>SubRack</string>
    </attribute>
    <widget class="QWidget" name="qwSubrack" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1191</width>
       <height>936</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="qtabLive">
    <attribute name="title">
     <string>Live Monitor</string>
    </attribute>
    <widget class="QWidget" name="qwLive" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1191</width>
       <height>936</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="qtabPlayback">
    <attribute name="title">
     <string>Playback</string>
    </attribute>
    <widget class="QLabel" name="play_qlabel_dataload">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>70</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="qwPlay" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1191</width>
       <height>936</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="qtabHelp">
    <attribute name="title">
     <string>Help</string>
    </attribute>
    <widget class="QLabel" name="qlabel_user_manual">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>591</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:6pt; font-weight:600; font-style:italic; color:#ffffff;&quot;&gt;space &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:48pt; font-weight:600; font-style:italic;&quot;&gt;SKA&lt;/span&gt;&lt;span style=&quot; font-size:48pt;&quot;&gt; in &lt;/span&gt;&lt;span style=&quot; font-size:48pt; font-weight:600; font-style:italic;&quot;&gt;LAB&lt;/span&gt;&lt;span style=&quot; font-size:48pt;&quot;&gt; project&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QWidget" name="qwpics_help" native="true">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>10</y>
       <width>491</width>
       <height>122</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="qlabel_sw_version">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>130</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Version: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="qlabel_sw_release">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>130</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Released on: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="qlabel_sw_author">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>130</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Author: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QTabWidget" name="tabHelp">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>1081</width>
       <height>742</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="qtab_intro">
      <attribute name="title">
       <string>Intro</string>
      </attribute>
      <widget class="QTextEdit" name="Help_Intro">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1081</width>
         <height>741</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(251, 249, 246);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="qtab_changelog">
      <attribute name="title">
       <string>ChangeLog</string>
      </attribute>
      <widget class="QTextEdit" name="Help_ChangeLog">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1081</width>
         <height>741</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(251, 249, 246);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="qtab_todo">
      <attribute name="title">
       <string>ToDo</string>
      </attribute>
      <widget class="QTextEdit" name="Help_ToDo">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1081</width>
         <height>741</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(251, 249, 246);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="qtab_note">
      <attribute name="title">
       <string>Notes</string>
      </attribute>
      <widget class="QTextEdit" name="Help_Notes">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1081</width>
         <height>741</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(251, 249, 246);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="qtab_troubleshooting">
      <attribute name="title">
       <string>Troubleshooting</string>
      </attribute>
      <widget class="QTextEdit" name="Help_TroubleShooting">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1081</width>
         <height>741</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(251, 249, 246);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
